src/index.js

```diff
@@ -4,12 +4,12 @@ import {BrowserRouter} from 'react-router-dom';
 
 import './index.css';
 import App from './App';
-// import AppV2 from './AppV2';
+import AppV2 from './AppV2';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
   <BrowserRouter>
-    {window.location.pathname.includes('v2') ? <App /> : <App />}
+    {window.location.pathname.includes('rc') ? <AppV2 /> : <App />}
```

Comment:
Why the change from v2 to rc?

Reply:
per our discussion on slack:

@mvines for routing question. it is ok if @sunny continue to develop his own components, i find it useful as a reference. i check them and learn how data is being requested and parsed. i believe it is quite overwork you do on your side but i am fine if you want to do it. it is like intermediate step between mvp and actual production code i am doing.

Michael Vines [12:40 PM]
@Manuel Calavera - between you and @sunny, just do what you both feel is the most efficient.

basically we have conflict on v2 route since @sunnygleason is doing his own react components on the same route.

Reply:
fine, this is gross but 🙈.  Next week we should try to get on the same non-v1 route if at all possible.  Maintaining 3 routes in the same tree is going to quickly turn into 🍝

Reply:
👍 agreed, let's get this sorted

Reply:
Hi there 👋 @manuel-calavera, I don't think I'm the @sunny you were looking for :)